test/parallel/test-force-repl.js

```diff
@@ -4,12 +4,21 @@ const assert = require('assert');
 const spawn = require('child_process').spawn;
 
 // Spawn a node child process in interactive mode (enabling the REPL) and
-// confirm the '> ' prompt is included in the output.
+// confirm the '> ' prompt and welcome message is included in the output.
 const cp = spawn(process.execPath, ['-i']);
 
 cp.stdout.setEncoding('utf8');
 
-cp.stdout.once('data', common.mustCall(function(b) {
-  assert.strictEqual(b, '> ');
-  cp.kill();
+let out = '';
+cp.stdout.on('data', (d) => {
+  out += d;
+});
+
+cp.stdout.on('end', common.mustCall(() => {
+  assert(out.startsWith('Welcome to Node.js'));
+  assert(out.includes('Type ".help" for more information.'));
+  assert(out.endsWith('> '));
+  console.log('ok');
```

Comment:
Nit: the `console.log` here should be removed and it would be possible to just check the whole string as in:

```
assert.strictEqual(out, `Welcome to Node.js ${process.version}.\n` +
                        'Type ".help" for more information.\n> ')
```

That way we also make sure that there is indeed a line break before `> ` (which I am not sure if it's currently there or not).